cleanlab/filter.py

```diff
@@ -368,18 +384,9 @@ class 0, 1, ..., K-1.
         label_issues_mask = find_predicted_neq_given(labels, pred_probs, multi_label=multi_label)
 
     # Remove label issues if given label == model prediction
-    if multi_label:
-        pred = _multiclass_crossval_predict(labels, pred_probs)
-        labels = MultiLabelBinarizer().fit_transform(labels)
-    else:
-        pred = pred_probs.argmax(axis=1)
+    pred = pred_probs.argmax(axis=1)
     for i, pred_label in enumerate(pred):
-        if (
-            multi_label
-            and np.all(pred_label == labels[i])
-            or not multi_label
-            and pred_label == labels[i]
-        ):
+        if not multi_label and pred_label == labels[i]:
```

Comment:
```suggestion
        if pred_label == labels[i]:
```

Reply:
You don't have to check for `multi_label` anymore right? Isn't all the multi_label code now moved to the helper `_find_label_issues_multilabel` function?